### PR TITLE
Update basics tutorials

### PR DESCRIPTION
**Pull request by stephanielee9**
Remove highlighted text and added install instructions

### CODE REVIEW

File: docs/source/get_started/basics/basics_css.ipynb [hunk 1]
```diff
@@ -37,12 +37,31 @@
    "id": "1f2c8de5",
```
Comment (vtomole) [inline docs/source/get_started/basics/basics_css.ipynb:37]: <p>Delete output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (stephanielee9) [reply docs/source/get_started/basics/basics_css.ipynb:37]: updated!

File: docs/source/get_started/basics/basics_qss.ipynb [hunk 2]
```diff
@@ -37,7 +37,17 @@
    "id": "1f2c8de5",
```
Comment (vtomole) [inline docs/source/get_started/basics/basics_qss.ipynb:37]: <pre class="ql-syntax" spellcheck="false">Line #1.    %pip install -q qiskit-superstaq[examples]
</pre><p>Some venvs might have qiskit-superstaq installed. We don't want to install them again on those in the 2 cell change to</p><p> </p><p><code></code>`python</p><p>try:</p><p>	import qiskit</p><p>	import qiskit_superstaq as qss </p><p>except ImportError:</p><p>    print("Installing qiskit-superstaq...")</p><p>    %pip install -q qiskit-superstaq[examples]</p><p>    print("Installed qiskit-superstaq")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Infleqtion/client-superstaq/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (stephanielee9) [reply docs/source/get_started/basics/basics_qss.ipynb:37]: <p>I'm trying to mirror the install instructions across the notebooks - do the other ones all need to change as well?</p>

Comment (vtomole) [reply docs/source/get_started/basics/basics_qss.ipynb:37]: > I'm trying to mirror the install instructions across the notebooks 
 
:+1: 

do the other ones all need to change as well? 

Let's leave this as-is and create an issue to change all notebooks to my suggested change.

**Review by vtomole — APPROVED** [review]
LGTM

**Pull request merged**